app/backend/db/database.py

```diff
@@ -0,0 +1,102 @@
+import os
+import json
+import names
+from contextlib import contextmanager
+
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+from db import models
+
+
+USERNAME = os.environ.get('ONE_REPORT_USERNAME', 'one_report')
+PASSWORD = os.environ.get('ONE_REPORT_PASSWORD', 'one_report')
+HOST = os.environ.get('ONE_REPORT_HOST', 'localhost')
+DB = os.environ.get('ONE_REPORT_DB', 'one_report')
+PORT = os.environ.get('ONE_REPORT_PORT', '5432')
+
+DATABASE_URI = f'postgres+psycopg2://{USERNAME}:{PASSWORD}@{HOST}:{PORT}/{DB}'
```

Comment:
you broke alembic import fix the import in alembic/env

Reply:
done